waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/PanopticOperationHandler.java

```diff
@@ -188,4 +192,38 @@
     // `Future.get` currently does not do that, but this is safe if implementation changes in the future
     return Math.max(1, requestTimeout + maxLatency);
   }
+
+  /**
+   * Implements {@link HMSHandler#get_all_functions()} over multiple metastores
+   *
+   * @return GetAllFunctionsResponse (db's from federated metastores will be prefixed if necessary)
+   */
+  public abstract GetAllFunctionsResponse getAllFunctions();
```

Comment:
I think this function and the one below could be moved on line 143, to be in the same bundle with the other functions that implement an `HMSHandler` function.
Not fussed about it, but it might be useful in the future when someone looks at this class :)

Reply:
yeah I'll move it a bit I think I'm gonna change this class a bit to make it more testable